moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -7,10 +7,14 @@ use_gazebo: false # Whether the robot is started in a Gazebo simulation environm
 command_in_type: "unitless" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
 scale:
   # Scale parameters are only used if command_in_type=="unitless"
-  linear:  0.4  # Max linear velocity. Meters per publish_period. Unit is [m/s]. Only used for Cartesian commands.
-  rotational:  0.8 # Max angular velocity. Rads per publish_period. Unit is [rad/s]. Only used for Cartesian commands.
-  # Max joint angular/linear velocity. Rads or Meters per publish period. Only used for joint commands on joint_command_in_topic.
+  linear:  0.4  # Max linear velocity. Unit is [m/s]. Only used for Cartesian commands.
```

Comment:
OK, will update it. Note that I was testing on a UR configuration, not here with the panda.

Reply:
I think these values were so large because the tutorial was publishing joystick commands of 0.2.

Changed the joystick command to 1.0 and changed this to 0.08. That makes a little more sense.

Reply:
New motion speed
![new_motion_speed](https://user-images.githubusercontent.com/11284393/144925418-0f65c73e-8780-443a-9307-c1ef97faeb37.gif)

Reply:
Is `joint4` the elbow on the Panda robot?

Reply:
Yeah -- confirmed with tf.

![elbow](https://user-images.githubusercontent.com/11284393/145100428-5e1bce61-f024-4da9-858a-072b5fbe639b.png)